

//Finally'
// the try catch construct has more clause that is finally

try{
    leta
}